api.md

```diff
@@ -243,36 +243,50 @@ distributed hypertables will not automatically use the newly added
 data node. For existing distributed hypertables to use added data
 nodes, use [`attach_data_node`](#attach_data_node).
 
-The function will attempt to bootstrap the data node by:
-1. Connecting to a specified postgres instance
-2. Creating the database that will serve as the new data node
-3. Loading the TimescaleDB extension on the new database
+If the data node already exists, the command will abort with either an
+error or a notice depending on `if_not_exists`.
+
+If `bootstrap` is true, the function will attempt to bootstrap the
+data node by:
+1. Connecting to the database given in `bootstrap_database`.
+2. Creating the database given in `database` that will serve as the
+   new data node.
+3. Loading the TimescaleDB extension in the new database.
+4. Set the distributed UUID of the database to to the UUID of the
```

Comment:
"to to" => "to"

Reply:
Fixed. There were another one that was not part of this commit, so fixed that as well.